### ISSUE TITLE
Current failures during npm test: Maximum call stack size exceeded

### ISSUE THREAD

**Issue by pascalgrimaud** [jhipster/jhipster-lite#737]
Probably related to a recent upgrade of one library with dependabot

```
[INFO] --- jacoco-maven-plugin:0.8.7:report (post-unit-test) @ beer ---
[INFO] Loading execution data file /tmp/jhlite/full-default/target/jacoco.exec
[INFO] Analyzed bundle 'beer' with 48 classes
[INFO] 
[INFO] --- frontend-maven-plugin:1.12.1:npm (front test) @ beer ---
[INFO] npm not inheriting proxy config from Maven
[INFO] Running 'npm run test' in /tmp/jhlite/full-default
[INFO] 
[INFO] > beer@0.0.0 test
[INFO] > jest src/test/javascript/spec
[INFO] 
[INFO] FAIL src/test/javascript/spec/common/primary/app/App.spec.ts
[INFO]   ● Test suite failed to run
[INFO] 
[INFO]     RangeError: /tmp/jhlite/full-default/App.component.ts: Maximum call stack size exceeded
[INFO] 
[INFO]       at Object.decode (node_modules/@jridgewell/src/sourcemap-codec.ts:41:23)
[INFO]       at new TraceMap (node_modules/@jridgewell/src/trace-mapping.ts:113:23)
[INFO]       at node_modules/@ampproject/src/build-source-map-tree.ts:80:20
[INFO]           at Array.map (<anonymous>)
[INFO]       at build (node_modules/@ampproject/src/build-source-map-tree.ts:50:36)
[INFO]       at node_modules/@ampproject/src/build-source-map-tree.ts:80:14
[INFO]           at Array.map (<anonymous>)
[INFO]       at build (node_modules/@ampproject/src/build-source-map-tree.ts:50:36)
[INFO]       at node_modules/@ampproject/src/build-source-map-tree.ts:80:14
[INFO]           at Array.map (<anonymous>)
[INFO] 
[INFO] Failed to collect coverage from /tmp/jhlite/full-default/src/main/webapp/app/common/primary/app/App.vue
[INFO] ERROR: /tmp/jhlite/full-default/App.component.ts: Maximum call stack size exceeded
[INFO] STACK: RangeError: /tmp/jhlite/full-default/App.component.ts: Maximum call stack size exceeded
[INFO]     at Object.decode (/tmp/jhlite/full-default/node_modules/@jridgewell/src/sourcemap-codec.ts:41:23)
[INFO]     at new TraceMap (/tmp/jhlite/full-default/node_modules/@jridgewell/src/trace-mapping.ts:113:23)
[INFO]     at /tmp/jhlite/full-default/node_modules/@ampproject/src/build-source-map-tree.ts:80:20
[INFO]     at Array.map (<anonymous>)
[INFO]     at build (/tmp/jhlite/full-default/node_modules/@ampproject/src/build-source-map-tree.ts:50:36)
[INFO]     at /tmp/jhlite/full-default/node_modules/@ampproject/src/build-source-map-tree.ts:80:14
[INFO]     at Array.map (<anonymous>)
[INFO]     at build (/tmp/jhlite/full-default/node_modules/@ampproject/src/build-source-map-tree.ts:50:36)
[INFO]     at /tmp/jhlite/full-default/node_modules/@ampproject/src/build-source-map-tree.ts:80:14
[INFO]     at Array.map (<anonymous>)
[INFO] 
[INFO] =============================== Coverage summary ===============================
[INFO] Statements   : 0% ( 0/4 )
[INFO] Branches     : 100% ( 0/0 )
[INFO] Functions    : 100% ( 0/0 )
[INFO] Lines        : 0% ( 0/3 )
[INFO] ================================================================================
[INFO] Jest: "global" coverage threshold for statements (100%) not met: 0%
[INFO] Jest: "global" coverage threshold for lines (100%) not met: 0%
[INFO] Test Suites: 1 failed, 1 total
[INFO] Tests:       0 total
[INFO] Snapshots:   0 total
[INFO] Time:        5.092 s
[INFO] Ran all test suites matching /src\/test\/javascript\/spec/i.
[INFO] ------------------------------------------------------------------------
[INFO] BUILD FAILURE
[INFO] ------------------------------------------------------------------------
[INFO] Total time:  01:30 min
[INFO] Finished at: 2022-02-15T17:53:41Z
[INFO] ------------------------------------------------------------------------
Error:  Failed to execute goal com.github.eirslett:frontend-maven-plugin:1.12.1:npm (front test) on project beer: Failed to run task: 'npm run test' failed. org.apache.commons.exec.ExecuteException: Process exited with an error: 1 (Exit value: 1) -> [Help 1]
Error:  
Error:  To see the full stack trace of the errors, re-run Maven with the -e switch.
Error:  Re-run Maven using the -X switch to enable full debug logging.
Error:  
Error:  For more information about the errors and possible solutions, please read the following articles:
Error:  [Help 1] http://cwiki.apache.org/confluence/display/MAVEN/MojoFailureException
Error: Process completed with exit code 1.
```

**Comment by jridgewell** [comment]
This is caused by https://github.com/babel/babel/issues/14273#issuecomment-1040869361, which is now fixed in v7.17.4

**Comment by pascalgrimaud** [comment]
Thanks a lot for this quick fix @jridgewell
